tests/foreman/cli/test_contentviews.py

```diff
@@ -184,19 +173,13 @@ def test_cv_create_cli_negative(self, test_data):
         system handles it gracefully
 
         """
+        try:
+            test_data['organization-id'] = make_org(cached=True)['id']
+        except CLIFactoryError as err:
+            self.fail(err)
 
-        org_obj = make_org(cached=True)
-
-        result = Org.info({'id': org_obj['id']})
-        self.assertEqual(result.return_code, 0, "Failed to create object")
-        self.assertEqual(
-            len(result.stderr), 0, "There should not be an exception here")
-
-        test_data['organization-id'] = org_obj['id']
-        result = ContentView.create(test_data)
-        self.assertNotEqual(result.return_code, 0)
-        self.assertGreater(len(result.stderr), 0,
-                           "There should be an exception here")
+        with self.assertRaises(CLIFactoryError):
+            make_content_view(test_data)
```

Comment:
The method ``make_content_view`` takes a dictionary but we're passing only an ID?